libraries/ms-common/src/can_fsm.c

```diff
@@ -53,6 +55,12 @@ static void prv_handle_rx(Fsm *fsm, const Event *e, void *context) {
     return;
   }
 
+  // Process bootloader messages
+  if(rx_msg.source_id == SYSTEM_CAN_DEVICE_BOOTLOADER) {
+    result = bootloader_can_receive(&rx_msg);
+  }
+  return;
```

Comment:
Not sure if this is the cause of the test error or not but the `return` should go inside the if statement, otherwise we're ignoring all normal CAN messages.